### PR TITLE
Initial package.json

### PR DESCRIPTION
**Pull request by LostButton**
I've made this package.json pr with all the previous PR's dependencies.

### CODE REVIEW

**Review by batunpc — APPROVED** [review]
Looks good!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my-photohub-1",
```
Comment (SerpentBytes) [inline package.json]: Do we need the "-1" in the `name`?

**Review by SerpentBytes — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (LostButton) [reply package.json]: i thought i changed that?

**Comment by SerpentBytes** [comment]
LGTM.

File: package.json (hunk 1 above)
Comment (SerpentBytes) [reply package.json]: Yeah, I was looking at an old commit.

**Review by SerpentBytes — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my-photohub",
+  "version": "1.0.0",
```
Comment (batunpc) [inline package.json:3]: Here let's start with version minor (i.e. "version": "0.1.0")

**Review by batunpc — CHANGES_REQUESTED** [review]

**Review by batunpc — APPROVED** [review]
Looks good!

**Review by SerpentBytes — APPROVED** [review]

**Pull request merged**